pymc/logprob/__init__.py

```diff
@@ -42,7 +42,6 @@
     transformed_conditional_logp,
 )
 
-# isort: off
```

Comment:
How is import sorting controlled now?

Reply:
Yeah, you are right! Let me try to solve this so that we do not ignore that many rules on the _init_.py files 🙏

Reply:
Fixed! I did not include isort in the config! Now https://github.com/pymc-devs/pymc/pull/7091/commits/6819090f9d2e65d80b6a257d63ea8bdea4900689 failed on purpose and https://github.com/pymc-devs/pymc/pull/7091/commits/c58501bed6eec60ded25671e28e627897d6bdc14 fixes it!